packages/page-accounts/src/Accounts/index.spec.ts

```diff
@@ -115,4 +212,70 @@ describe('Accounts page', () => {
 
     return balanceOf(amountInt.toString() + decimalsPadded);
   };
+
+  const accountsWithStaking = [
+    {
+      address: '5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy',
+      balance: {
+        availableBalance: balance(400),
+        freeBalance: balance(500),
+        lockedBalance: balance(70)
+      },
+      staking: {
+        redeemable: balance(4000),
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        stakingLedger: {
+          active: {
+            unwrap: () => balance(30)
+          }
+        } as any,
+        unlocking: [
+          {
+            remainingEras: new BN('1000000000'),
+            value: balance(200)
+          },
+          {
+            remainingEras: new BN('2000000000'),
+            value: balance(300)
+          },
+          {
+            remainingEras: new BN('3000000000'),
+            value: balance(400)
+          }
+        ]
+      }
+    },
+    {
+      address: '5HGjWAeFDfFCWPsjFQdVV2Msvz2XtMktvgocEZcCj68kUMaw',
+      balance: {
+        availableBalance: balance(600),
+        freeBalance: balance(200),
+        lockedBalance: balance(750),
+        reservedBalance: balance(150)
+      },
+      staking: {
+        redeemable: balance(3000),
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        stakingLedger: {
+          active: {
+            unwrap: () => balance(60)
```

Comment:
Should be fairly easy to create an `Option` instance. Sth along the lines of:
`new TypeRegistry().createType('Option<Balance>', new TypeRegistry().createType('Balance', new BN(number)));`
or https://github.com/polkadot-js/api/blob/master/packages/api-derive/test/bountyFactory.ts#L30.

Basically the big question is whether we keep creating the `TypeRegistry` for each creation method or reuse a single instance across all tests.

Reply:
Done. I did not create a `TypeRegistry` just yet because we create new ones in other places too